lib/tasks/coverage_report.rake

```diff
@@ -7,17 +7,15 @@ namespace :coverage do
   desc "Collates all result sets generated by the different test runners"
   task :report do
     require 'simplecov'
-    require 'simplecov_json_formatter'
-    unless ENV['CI']
-      puts 'This task is intended to be run only on the CI.'
-      return
-    end
-    SimpleCov.collate(Dir["coverage*/.resultset.json"]) do
-      formatter SimpleCov::Formatter::MultiFormatter.new([
-        SimpleCov::Formatter::JSONFormatter,
-        SimpleCov::Formatter::HTMLFormatter
-      ])
```

Comment:
My understanding is that these were used while debugging the original SimpleCov results and we don't need them for anything now. HTML formatter is the default, so we'll still get a nice `index.html` file to search.

Reply:
I believe that's accurate based on what I remember about these